File: src/codes/Users_Manual/geometry_0.rs
# use autd3::prelude::*;
# 
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let _autd = 
Controller::builder([
        AUTD3::new(Point3::origin()), 
        AUTD3::new(Point3::new(AUTD3::DEVICE_WIDTH, 0., 0.))
    ])
#    .open(autd3::link::Nop::builder())?;
# Ok(())
# }

File: src/codes/Users_Manual/geometry_0.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~autd3::link::Nop link;
autd3::Controller::open({autd3::AUTD3{
                             .pos = autd3::Point3::origin(),
                             .rot = autd3::Quaternion::Identity(),
                         },
                         autd3::AUTD3{
                             .pos = autd3::Point3(autd3::AUTD3::DEVICE_WIDTH, 0,
                                                  0),
                             .rot = autd3::Quaternion::Identity(),
                         }},
                        std::move(link));
//~return 0; }